tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -101,8 +105,16 @@ class CloudGov(Common):
     AWS_S3_STATICFILES_BUCKET_NAME = s3_staticfiles_creds['bucket']
     AWS_S3_STATICFILES_ENDPOINT = f'https://{s3_staticfiles_creds["endpoint"]}'
     AWS_S3_STATICFILES_REGION_NAME = s3_staticfiles_creds['region']
+
+    # TODO This was supose to be able to handle our s3 prefixes in 1 line, but
+    # we rolled a custom solution to multiple buckets with boto3 django extension
+    # instead of prefixing then. We'd like to be able to use this, as it is simpler
+    # AWS_LOCATION = cloudgov_name
```

Comment:
Maybe this wasn't working as a result of a typo I made before that made cloudgov_name an object?

Reply:
Were you able to resolve this? I feel I'm lacking context too.

Reply:
It has to do with the `APP_NAME` and inheritance we are doing to use multiple s3's.